editor.planx.uk/src/ui/DateInput.tsx

```diff
@@ -19,117 +18,101 @@ export interface Props {
 const INPUT_DATE_WIDTH = "65px";
 const INPUT_YEAR_WIDTH = "80px";
```

Comment:
Can I ask you to up this to `84px` please? Since the font size changes were actioned the placeholder text is cut off a little.